packages/react-ui-kit/src/Form/Select.tsx

```diff
@@ -31,16 +31,17 @@ import type {Theme} from '../Layout';
 import InputLabel from './InputLabel';
 
 export type Option = {
-  value: string | number;
+  value: string;
```

Comment:
the value *will* sometimes be a number

Reply:
in what case do we return a number? when we talk about inputs the input value will always be a string

Reply:
cipher suites need to be a number for the BE to work
<img width="495" alt="Screenshot 2022-07-29 at 13 49 15" src="https://user-images.githubusercontent.com/37285713/181752687-32a86a2b-12c7-4d87-be77-5e444af71200.png">
<img width="796" alt="Screenshot 2022-07-29 at 13 49 38" src="https://user-images.githubusercontent.com/37285713/181752693-bbdd20e0-08f5-40a1-a760-454d91b0f349.png">

Reply:
But that gets done from the onChange handler right? The option will and the onChange on the select are supposed to work with strings

Reply:
sorry i missed this mesage but yea thats fine also